mobile/src/viewModels/HomeViewModel.ts

```diff
@@ -108,7 +108,13 @@ export default class HomeViewModel {
                 <IFinishQolTipItem>{
                     id: 'finish-qol',
                     type: 'finish-qol',
-                    title: 'Tap to continue your check-in!',
+                    title: 'Tap to continue your QoL Survey!',
+                },
+                // Currently always shows 'daily' check-in as this feature has not yet been implemented
+                <ICheckInTipItem>{
+                    id: 'check-in',
+                    type: 'check-in',
+                    title: AppViewModel.Instance.CreateCheckIn.question || 'Create a new check-in!',
```

Comment:
Should 'check-in' here also be changed to 'Survey'?

Reply:
I don't think so, that for the daily check in

Reply:
Gotcha